### PR TITLE
[MINOR][DOC] Documentation on JVM options for SBT

### PR DESCRIPTION
**Pull request by da-liii**
## What changes were proposed in this pull request?

Documentation and .gitignore

## How was this patch tested?

Manual test that SBT honors the settings in .jvmopts if present

### CODE REVIEW

File: docs/building-spark.md [hunk 1]
```diff
@@ -138,6 +138,13 @@ To avoid the overhead of launching sbt each time you need to re-compile, you can
 in interactive mode by running `build/sbt`, and then run all build commands at the command
 prompt.
 
+### Setting up SBT's Memory Usage
+Configure the JVM options for SBT in `.jvmopts`, for example:
+
+    $ cat .jvmopts                                                                                                                   [21:18:11]
+    -Xms512M
```
Comment (srowen) [inline docs/building-spark.md]: I doubt we need to suggest `-Xms`; for simplicity, remove it? Also let's make it consistent with the Maven opts above: `-Xmx2g`?

I'm not sure what [21:18:11] is; we don't need to show `cat`. But this does need to say that the file should live in the project root and show its one line of content.

**Review by srowen — CHANGES_REQUESTED** [review]

File: docs/building-spark.md (hunk 1 above)
Comment (sadhen) [reply docs/building-spark.md]: Sorry, I committed my console time.

**Comment by sadhen** [comment]
@srowen I've updated the doc. Please tell me if there are any errors. (A bit embarrassing 囧  )

**Review by srowen — APPROVED** [review]

**Comment by dongjoon-hyun** [comment]
Hi, @sadhen . Could you update the section, `How was this patch tested?`?
```
## How was this patch tested?
None.
```

**Comment by nchammas** [comment]
Is there any reason not to just check in a `.jvmopts` file that captures these settings, and remove the corresponding entry from `.gitignore`? Seems like that would be better, since it saves contributors from a manual step. If appropriate, I can submit a PR for this.

**Pull request closed without merging**